access_rights.md

```diff
@@ -0,0 +1,157 @@
+# Grin Access Rights
+
+Last updated: 25/05/21
+
+## Table of Contents
+1. [Introduction](#introduction)
+1. [Crates.io](#crates.io)
+1. [Discord](#discord)
+1. [Email](#email)
+1. [Forum](#forum)
+1. [Funds](#funds)
+1. [Github](#github)
+1. [Keybase](#keybase)
+1. [Mailing List](#mailing-list)
+1. [Snapcraft](#snapcraft)
+1. [Telegram](#telegram)
+1. [Twitter](#twitter)
+1. [Website](#website)
+
+
+## Introduction
+
+This document tracks and summarizes all the owner of key ressources of the Grin project. Some of the data such as the multisignature owners are anonymized.
+
+## Crates.io
+
+Ability to publish to [grin crates](https://crates.io/search?q=grin) to crates.io as authors "Grin Developers" with email mimblewimble@lists.launchpad.net:
+- [quentinlesceller](https://github.com/quentinlesceller)
+- [yeastplume](https://github.com/yeastplume)
+
+## Discord
+
+Owner of Discord "Grin Community":
+- [mcm-mike](https://github.com/mcm-mike)
+
+## Email
+
+* Ownership of grincouncil@protonmail.com:
+   - X
+   - X
+
+* Members of grinmods@googlegroups.com
+_As per [CoC]( https://github.com/mimblewimble/grin/blob/master/CODE_OF_CONDUCT.md)_
+
+## Forum 
+
+Currently hosted on Discourse at forum.grin.mw. Paid monthly by [quentinlesceller](https://github.com/quentinlesceller).
+
+### Administrators
+
+- [lehnberg](https://github.com/lehnberg) 
+- [quentinlesceller](https://github.com/quentinlesceller)
+- [tromp](https://github.com/tromp)
+
+
+### Moderators
+
+Including administrators:
+
+- [jaspervdm](https://github.com/jaspervdm)
+- [lehnberg](https://github.com/lehnberg)
+- [quentinlesceller](https://github.com/quentinlesceller)
+- [tromp](https://github.com/tromp)
+- [yeastplume](https://github.com/yeastplume)
+
+## Funds 
+
+### Bitcoin donation addresses
+
+Bitcoin addresses 39hgKkhfPFpwKRQ6yARGwU2uuVay69NKu4 and bc1qecw954uappnwhle6tmafqskn6vxmpr6lmmuwfwldphfxplcshgnssy9kmx adverstised on https://github.com/mimblewimble/grin-pm/blob/master/financials/addresses.md.
+Both of these addresses are x-of-y multisignatures (both numbers are hidden for privacy sake, y is a subset of the Grin council).
+
+### Grin donation address
+
+Hosted at https://council-donations.yeastplume.org advertised on https://github.com/mimblewimble/grin-pm/blob/master/financials/addresses.md#grin:
+- yeastplume
+
+
+
+
+## GitHub
+
+### Organization Members, Status and Teams Membership
+
+| Username | Status | Teams |
```

Comment:
My understanding is that there is no hierarchy among github owners. This means that we could remove inactive contributors from the list of owners, including @ignopeverell. I think this would be wise. It would be trivial to add them back if they ever returned.

It's probably easiest if I just make a specific proposal, and then everyone can critique it and propose changes:

* Promote @tromp, @joltz, and @phyro to owner (assuming each plans on continuing to being active in the community for the foreseeable future).

* Grant me merge access for grin & grin-wallet so that I may merge non-controversial maintenance PRs.

* Remove @antiochp, @garyyu, @hashmap, @ignopeverell, @jaspervdm, @nijynot, @lehnberg, and @yeastplume. We would gladly restore the privileges for every one of them if they choose to return, but for security purposes, it makes little sense to grant privs to so many inactive contributors.

Reply:
Agreed on additional ownership, but for ownership removal, I'd grant a 6 month inactivity window out of courtesy. That would be keep @antiochp, @lehnberg and @jaspervdm on a little longer. The others I think have not been active in the last 6 months, and can be removed.

Reply:
I agree with both of you. Would make sense to me to do the following:
- Promote @tromp, @j01tz, and @phyro to owner. If they want to.
- Remove @garyyu, @hashmap, @ignopeverell and @yeastplume.

Reply:
I'm fine with ownership.

Reply:
I'm also fine. Did you forget nijynot on the remove list or were they left in on purpose?

Reply:
I can take the responsibility as well. I don't have a lot of time these days but will always make time for Grin as needed.

Reply:
> I'd grant a 6 month inactivity window out of courtesy.

As mentioned in keybase, I don't think it's discourteous to remove them while they're inactive. We would gladly add them back if they returned. But giving privileges to people who don't need them is just bad security policy.

I think it's clear that Lehnberg and Jasper have little need for ownership rights, and the fact that Antioch has not responded since you asked him about his future contribution plans, is a pretty big indicator he's also not going to be around (in addition to not being active in nearly 2 months, of course).

Reply:
> My understanding is that there is no hierarchy among github owners. This means that we could remove inactive contributors from the list of owners, including @ignopeverell. I think this would be wise. It would be trivial to add them back if they ever returned.
> 
> It's probably easiest if I just make a specific proposal, and then everyone can critique it and propose changes:
> 
> * Promote @tromp, @j01tz, and @phyro to owner (assuming each plans on continuing to being active in the community for the foreseeable future).
> * Remove @antiochp, @garyyu, @hashmap, @ignopeverell, @jaspervdm, @nijynot, @lehnberg, and @yeastplume. We would gladly restore the privileges for every one of them if they choose to return, but for security purposes, it makes little sense to grant privs to so many inactive contributors.
> 
> EDIT: Removed request to grant myself merge privileges. It's unnecessary if @phyro and others are given ownership of the org.